desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
So, I think the following rule changes may get the job done (I wish Chrome made it easy to a diff of style edits):

```
.artwork-section {
  /* margin: 10px 0 60px 0; */
  margin: 10px 0 40px 0;
}

#react-mount-current-auctions {
    /* margin-top: -60px; */
    margin-bottom: 60px;
}
```

Note: I would probably style the `.artwork-current-auctions` section tag by class, rather than styling the mount element by ID. Even more so in this case, since the mount element is unconditionally present.

```
/* ...continued */

.artwork-current-auctions__header-area h3 {
  /* margin-top: 20px */
}
```

Note: in BEM, every unconditional style can be done by a simple class selector. I would have given the `h3` tag an element class, like `.artwork-current-auctions__header`, and avoided styling by element. This would decouple the stylesheet from the arbitrary choice of tag. It also didn't seem ideal to be adding margin to the header tag to create spacing, rather than styling the block's root element.

```
/*... continued */

.artwork-artist-related-rail {
    /* padding-top: 15px; */
    /* margin: 10px 0 60px 0; */
    /* border-top: 1px solid #e5e5e5; */
}
```

**With current auctions shown:**

<img width="793" alt="image" src="https://cloud.githubusercontent.com/assets/1256382/24439079/f57545d4-1419-11e7-9eff-f4bd8be8a569.png">

**Without:**

<img width="780" alt="image" src="https://cloud.githubusercontent.com/assets/1256382/24439074/e907a56c-1419-11e7-805f-d11046762faf.png">